burn-import/src/onnx/op_configuration.rs

```diff
@@ -81,6 +82,41 @@ pub fn conv2d_config(curr: &Node) -> Conv2dConfig {
     .with_padding(padding)
 }
 
+/// Create a MaxPool2dConfig from the attributes of the node
+pub fn max_pool2d_config(curr: &Node) -> MaxPool2dConfig {
+    let mut channels: i64 = 1;
+    let mut kernel_shape = Vec::new();
+    let mut strides = Vec::new();
+    let mut pads = Vec::new();
+
+    for (key, value) in curr.attrs.iter() {
+        match key.as_str() {
+            "channels" => attr_value_i64(value, &mut channels),
+            "kernel_shape" => attr_value_vec_i64(value, &mut kernel_shape),
+            "strides" => attr_value_vec_i64(value, &mut strides),
+            "pads" => attr_value_vec_i64(value, &mut pads),
+            _ => {}
+        }
+    }
+
+    let padding = if pads.iter().all(|&x| x == 0) {
+        MaxPool2dPaddingConfig::Valid
+    } else {
+        todo!("max_pool2d: padding({pads:?}) is not fully supported");
+    };
+
+    if strides.iter().all(|&x| x != 1) {
+        todo!("max_pool2d: strides({strides:?}) are not fully supported");
```

Comment:
Sure! Should I simply remove this condition? Because it only checks that strides values are **all** equals to 1.

Reply:
What you need to do is use the attributes `strides`, `padding`, `groups` and `dilations` and add them to the `Conv2dConfig`. I don't think you need to change the codegeneration part, since it's based on the config.